Makefile

```diff
@@ -26,13 +26,19 @@ $(build)/$(function)/message.c:
 	mkdir -p $(build)/$(function)
 	cp $(function)/src/main/proto/* $(build)/$(function)
 
-$(build)/$(function)/$(wasm): $(build)/$(function)/message.c $(build)/lib/libprotoc.a 
+$(build)/$(function)/$(wasm): $(build)/$(function)/message.c $(build)/lib/libprotoc.a
```

Comment:
This is a manual target (not included in `ALL` or `function`) to easily compile+generate the `CloudEvent` proto C files. I mentioned it in the main README as well.